scripts/host_explore.py

```diff
@@ -49,63 +49,71 @@ def check_output(command, dir=None):
 
     return output
 
+
+def get_cxx_compiler(tgtType):
+    """Parse the config options prefixed by `tgtType` and determine the name of
+    the compiler for that target type, as well as whether it is a cross
+    compiler.
+    """
+    cross_compiler = False
+
+    if get_config_bool(tgtType + "_TOOLCHAIN_GNU"):
+        prefix = get_config_string(tgtType + "_GNU_TOOLCHAIN_PREFIX")
+        # If the prefix ends with the path separator, it is being used as
+        # alternative to adding the compiler to $PATH, and isn't specifying a
+        # cross-compilation prefix.
+        if prefix and not prefix.endswith(os.sep):
+            cross_compiler = True
+        cxx = prefix + get_config_string("GNU_CXX_BINARY")
+    elif get_config_bool(tgtType + "_TOOLCHAIN_CLANG"):
+        if get_config_string(tgtType + "_CLANG_TRIPLE"):
+            cross_compiler = True
+        cxx = get_config_string("CLANG_CXX_BINARY")
+    elif get_config_bool(tgtType + "_TOOLCHAIN_ARMCLANG"):
+        if tgtType != "HOST":
+            cross_compiler = True
+        cxx = get_config_string("ARMCLANG_CXX_BINARY")
+
+    return cxx, cross_compiler
+
+
+def stl_rpath_ldflags(tgtType):
+    """If the toolchain is generating code that can run on the host, add the
+    C++ STL library to the linker rpath so it can be found when the binary is
+    executed.
+    """
+    cxx, cross_compiler = get_cxx_compiler(tgtType)
+
+    if cross_compiler:
+        return []
+
+    stl_path = check_output([cxx, "-print-file-name=libstdc++.so"])
+    stl_dir = os.path.dirname(stl_path)
+    ld_library_path = os.getenv("LD_LIBRARY_PATH", "")
```

Comment:
I'll suggest to inform via key name about the second argument functionality (which is fallback output)

```suggestion
    ld_library_path = os.getenv("LD_LIBRARY_PATH", default="")
```

Reply:
Sure, I can do that.